### PR TITLE
Service borg updates: Part 1 - The debloatening

### PR DESCRIPTION
**Pull request by Majkl-J**
## About The Pull Request

I was let too close to cyborg code with #27227

Now I am out for blood.

This PR guts the default modules for the service borg down quite significantly, moving them all to a very cheap new module.
image

Also opts to use the tg cooking tool, removing other set of duplicate items.

This should hopefully make having 4 layers of modules an optional decision for borgs, instead of a requirement. This PR is also a prerequisite for the upcoming service borg botany update, which will introduce the botany module.

## How This Contributes To The Skyrat Roleplay Experience

If you want to play service for the cooking and not for the large set of usually unused artistic tools, you now have the option to.

Your screen will no longer be an obstructed mess.

## Proof of Testing

See the above screenshot. I am absolutely certain the other code works as I have tested the for loops extensively and plan on bringing them to TG because they're really useful :3

## Changelog
:cl:
qol: Debloated the service borg module count
balance: Moved service borg art and music supplies to the new artistic module
fix: Fixed the dual cooking tools on service borgs, opting to keep tg ones.
spellcheck: Fixed lowercase on a few borg module recipes
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/borgs/code/robot_upgrade.dm [hunk 1]
```diff
@@ -263,7 +263,7 @@
 	)
 
 /obj/item/borg/upgrade/cargo_tele
-	name = "borg cargo teleporter module"
+	name = "Cargo teleporter module"
```
Comment (jjpark-kb) [inline modular_skyrat/modules/borgs/code/robot_upgrade.dm]: we don't capitalize these

File: modular_skyrat/modules/borgs/code/robot_upgrade.dm [hunk 2]
```diff
@@ -294,7 +294,7 @@
 		cyborg.model.remove_module(locate_tele, TRUE)
 
 /datum/design/borg_upgrade_forging
-	name = "borg forging module"
+	name = "Forging module"
```
Comment (jjpark-kb) [inline modular_skyrat/modules/borgs/code/robot_upgrade.dm:297]: we don't capitalize these

File: modular_skyrat/modules/borgs/code/robot_upgrade.dm [hunk 3]
```diff
@@ -356,6 +356,60 @@
 	if(locate_forge)
 		cyborg.model.remove_module(locate_forge, TRUE)
 
+/*
+* SERVICE CYBORG UPGRADES
+*/
+
+/datum/design/borg_upgrade_artistic
+	name = "Artistic module"
```
Comment (jjpark-kb) [inline modular_skyrat/modules/borgs/code/robot_upgrade.dm:364]: we don't capitalize these

File: modular_skyrat/modules/borgs/code/robot_upgrade.dm (hunk 1 above)
Comment (Maia-J) [reply modular_skyrat/modules/borgs/code/robot_upgrade.dm]: Ah I meant to capitalize the design, woops

**Pull request merged**